eng/packages/http-client-csharp/generator/Azure.Generator/test/Azure.Generator.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<!-- This is a workaround on the target frameworks, because in our build system, we requires to have net8.0, net6.0 and net462 -->
+		<TargetFrameworks>net8.0</TargetFrameworks>
+		<RequiredTargetFrameworks>net8.0</RequiredTargetFrameworks>
```

Comment:
We have [IsGeneratorLibrary](https://github.com/Azure/azure-sdk-for-net/blob/e74aac0e3884b42cd51520355613e10f957aedd5/eng/Directory.Build.Common.props#L14C6-L14C25) defined
It might be better to update the condition [here](https://github.com/Azure/azure-sdk-for-net/blob/e74aac0e3884b42cd51520355613e10f957aedd5/eng/Directory.Build.Common.props#L140) to include generator test project.

Reply:
fixed